library/include/ck/library/reference_tensor_operation/cpu/reference_batchnorm_backward_nhwc_c.hpp

```diff
@@ -0,0 +1,285 @@
+#ifndef REFERENCE_BATCHNORM_BACKWARD_NHWC_C_HPP
+#define REFERENCE_BATCHNORM_BACKWARD_NHWC_C_HPP
+
+#include <iostream>
+#include <sstream>
+#include <algorithm>
+#include "device_batchnorm_backward.hpp"
+
+namespace ck {
+namespace tensor_operation {
+namespace host {
+
+template <typename InOutDataType, typename AccDataType>
+struct ReferenceBatchNormBwd_Input_N_H_W_C_Output_C : public device::DeviceBatchNormBwd
```

Comment:
Can this be simplified into chaining respective backward NHWC reference op (implemented) with a standalone reference batchnorm op? I am trying to understand if code reuse is possible.

Reply:
Do you mean a generic BachNorm backward implementation, rather than specific for NHWC ?